src/components/home/Landing.jsx

```diff
@@ -35,8 +35,8 @@ const Landing = () => {
         </div>
       </div>
 
-      <div className="flex   items-center relative w-screen lg:-top-36 lg:-right-24 md:-top-52 md:-right-12 sm:-top-80 sm:-right-16 ">
-        <div className="flex   outline outline-rishi-orange outline-3 rounded-full  ">
+      <div className="flex items-center relative w-screen lg:-top-36 lg:-right-24 md:-top-52 md:-right-12 sm:-top-80 sm:-right-16 ">
+        <div className="flex outline outline-rishi-orange outline-3 rounded-full  ">
           <Image
```

Comment:
<img width="825" alt="image" src="https://github.com/acm-ucr/rishi-website/assets/90938120/59c4c31e-6173-497e-ab32-05cbef97cbed">

can you also move the image down on tablet screen sizes